ci/citest/acceptance-test/multibox/ind-steps/step-qemu-kvm/postconfigure.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+procc_type=
+case ${ACCEL_TYPE} in
+    vmx) procc_type="intel" ;;
+    svm) procc_type="amd" ;;
+    *) echo "Invalid hardware acceleration" ; exit 255 ;;
```

Comment:
Bashsteps sends exit code 255 when it exits so the loop that iterates the steps listens for it, since this is not inside a step I forced the exit code, maybe these type of checks should be put inside a step as well though.